package.json

```diff
@@ -34,8 +34,7 @@
     "turbo": "^1.6.2"
   },
   "engines": {
-    "node": "18.x",
-    "pnpm": "7.x"
-  },
-  "packageManager": "pnpm@7.25.1"
+    "node": "18.15.0",
+    "pnpm": "7.30.3"
```

Comment:
IMO the engines shouldn't be specific unless you don't support a specific version
I'm also wondering why you added engines on the other `package.json` files ?
It should only be needed where you do an install and we only do that in the root folder of the monorepo